src/scss/_back-to-top.scss

```diff
@@ -0,0 +1,36 @@
+#return-to-top {
+  position: fixed;
+  bottom: 1.25em; //20px
+  right: 1.25em; //20px
+  background: rgb(0, 0, 0);
```

Comment:
Ok, I was able to pull down your project and view it in my browser (thanks 100% to Zastrow). The code looks good as far as I can. Love the smooth scrolling you have, I might use some of that if you don't mind. The "back to top" button might need to have it's z-index adjusted, it seems to be going behind a few items on your page.